regtest/.env

```diff
@@ -1,20 +1,4 @@
-COMPOSE_PROFILES=default
-SCRIPTS_IMAGE=boltz/scripts:latest
-LND_IMAGE=boltz/lnd:latest
-CLN_IMAGE=boltz/c-lightning-plugins:latest
-POSTGRES_IMAGE=postgres:14-alpine
-BOLTZ_BACKEND_IMAGE=boltz/boltz:latest
-BOLTZ_WEBAPP_IMAGE=boltz/web-app:latest
-BOLTZ_CLIENT_IMAGE=boltz/boltz-client:latest
-FOUNDRY_IMAGE=boltz/foundry:latest
-BITCOIN_CORE_IMAGE=boltz/bitcoin-core:latest
-ELEMENTS_IMAGE=boltz/elements:latest
-ELECTRS_IMAGE=boltz/electrs:latest
-ESPLORA_IMAGE=boltz/esplora:latest
-RIF_RELAY_IMAGE=boltz/rif-relay:latest
-NGINX_IMAGE=nginx:alpine
-NBXPLORER_IMAGE=nicolasdorier/nbxplorer:2.5.4
 MEMPOOL_WEB_IMAGE=mempool/frontend:latest
 MEMPOOL_API_IMAGE=mempool/backend:latest
 MEMPOOL_DB_IMAGE=mariadb:10.5.21
-RT_SYNC_IMAGE=hydrayse/data-sync:latest
+RT_SYNC_IMAGE=danielgranhao/data-sync:latest
```

Comment:
There a specific reason for this change?

Reply:
It updates the data sync service to a more recent version (to include https://github.com/breez/data-sync/pull/4). Once https://github.com/breez/data-sync/pull/18 is merged, we can finally set it to the proper value. I've just pinged @hydra-yse to see if we can do it now.